backend/src/Squidex/Config/Authentication/OidcHandler.cs

```diff
@@ -26,14 +26,18 @@ public override Task TokenValidated(TokenValidatedContext context)
 
             if (!string.IsNullOrWhiteSpace(options.OidcRoleClaimType) && options.OidcRoleMapping?.Count >= 0)
             {
-                var role = identity.FindFirst(x => x.Type == options.OidcRoleClaimType)?.Value;
+                var roles = identity
+                    .FindAll(c => c.Type == options.OidcRoleClaimType)
```

Comment:
Just a very small change request:

This can be changed to FindAll(options.OidcRoleClaimType)

Reply:
or even

options.OidcRoleMapping.Where(r => identity.HasClaim(options.OidcRoleClaimType, r.Key))

Reply:
good catch, thanks